tensornetwork/backends/symmetric/decompositions.py

```diff
@@ -193,14 +193,13 @@ def svd_decomposition(
       discarded_singvals > 0.0]
 
 
-def qr_decomposition(bt, tensor: BlockSparseTensor,
-                     split_axis: int) -> Tuple[Tensor, Tensor]:
+def qr(bt, tensor: BlockSparseTensor, pivot_axis: int) -> Tuple[Tensor, Tensor]:
```

Comment:
can you add the non_negative_diagonal here and raise an error if its True?

Reply:
This happens at the backend level